cmd/kava/cmd/iavlviewer/root.go

```diff
@@ -68,11 +71,9 @@ func readTree(db dbm.DB, version int, prefix []byte) (*iavl.MutableTree, error)
 	if len(prefix) != 0 {
 		db = dbm.NewPrefixDB(db, prefix)
 	}
+	dbt := wrapper.NewCosmosDB(db)
```

Comment:
Can we make sure we are closing all connections to prevent resource leaks.

Reply:
great callout, though not strictly necessary in this context. because of the short-lived nature of `iavlviewer` as a CLI command it will be safely closed after the cmd.

in an effort to keep the diff with `release/v0.26.x` as small as possible, i am going to forego explicitly closing the tree in this brnach. but i've opened a pr on master to bring it into future releases: https://github.com/Kava-Labs/kava/pull/2000